tests/wificountry.bats

```diff
@@ -1,12 +1,22 @@
 #!/usr/bin/env bats
 load test-helper
 
+@test "$clinom wificountry" {
+  run "${clicmd}" wificountry
+  assert_success && assert_output -p 'country='
+}
+
 @test "$clinom wificountry US" {
   run "${clicmd}" wificountry US
   assert_success && assert_output -p 'country has been set'
 }
 
-@test "$clinom wificountry" {
-  run "${clicmd}" wificountry
-  assert_success && assert_output -p 'country='
-}
\ No newline at end of file
+@test "$clinom wificountry foo (invalid country code)" {
+  run "${clicmd}" wificountry foo
+  assert_output -p 'invalid country code'
+}
+
+@test "$clinom wificountry us (lower case country code)" {
+  run "${clicmd}" wificountry us
+  assert_success && assert_output -p 'country has been set'
+}
```

Comment:
My test fails at wificountry us (lower case country code):

![image](https://user-images.githubusercontent.com/61807745/114236664-cce86c80-9936-11eb-80e5-1d2f8f09a345.png)

Maybe it's case sensitive?

Reply:
That fail won't be fixed until this issue is resolved: https://github.com/treehouses/cli/issues/2167